### DISCUSSION TITLE
How to contribute a fix

### DISCUSSION THREAD

**Discussion by lucky-child** [OpenVoiceOS/ovos-buildroot#203] [Q&A] [answered]
Hello, how can I contribute a little fix that I made in my locale repository. I know how to use git but I'm not familiar with OpenVoiceOS@github. I guess that a simple git push will not work ;-)

**Comment by ChanceNCounter** [comment, marked as answer]
We should produce a more comprehensive contribution guide. The gist:

1. Fork the repository here on GitHub. This will give you your own instance to which you can push.
2. Push =P
3. After you push, GitHub will helpfully pop a message in your terminal inviting you to open a pull request, with a link right there to do it. If that doesn't happen, you can do it from the "pull requests" tab at our repository, or by navigating to your own repository (where that same "click here to open a PR" thing should come up graphically)

That's all you need to do to get as far as code review. With extremely few exceptions, our code is under permissive licenses whose terms are quite clear, and as we understand them, there's no need for you to sign or do anything. As long as you're offering your own code, or can cite another permissively-licensed source with proper credit, your contributions are welcome.

**Reply by lucky-child** [reply]
Will try ...
Thanks for your help :-)
